src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -154,13 +154,13 @@
     <value>The default value of an uninitialized enumeration, just as other value types, is zero. A nonflags-attributed enumeration should define a member by using the value of zero so that the default value is a valid value of the enumeration. If an enumeration that has the FlagsAttribute attribute applied defines a zero-valued member, its name should be ""None"" to indicate that no values have been set in the enumeration.</value>
   </data>
   <data name="EnumsShouldHaveZeroValueMessageFlagsRename" xml:space="preserve">
-    <value>In enum {0}, change the name of {1} to 'None'</value>
+    <value>In enum '{0}', change the name of '{1}' to 'None'</value>
```

Comment:
💡 It may be good to lock `None`

```suggestion
    <value>In enum '{0}', change the name of '{1}' to 'None'</value>
    <comment>{Locked="None"}</comment>
```

This would apply to all identifiers in resources representing API names.

We can approach this in two ways:

1. Have a first pass that adds `'` around true placeholders (e.g. `'{0}'`) and a second pass that addresses identifiers (e.g. `'None'` or `'FlagsAttribute'`)
2. Have one pass that covers both cases for a single resource file

Whichever way we go, the locking for names should be done in the pass that updates API names.